test/e2e/case17_gatekeeper_sync_test.go

```diff
@@ -88,6 +92,9 @@ var _ = Describe("Test Gatekeeper ConstraintTemplate and constraint sync", Order
 	})
 
 	AfterAll(func() {
+		if gkSyncDisabled {
```

Comment:
Is this required in the `AfterAll` or just in the `BeforeAll`?

Reply:
I'll try it again to be sure, but I think it was needed in both places since `Skip()` didn't skip `AfterAll()`.

Reply:
You're right--it's only needed in `BeforeAll()`. Must have been failing for a different reason before.